script/signin.js

```diff
@@ -0,0 +1,77 @@
+import { inputEmail, inputPassword, eyeIcon, form } from "./tags.js";
+import { createTag, removeTag, checkEmail } from "./function.js";
+import userData from "./emailData.js";
+let emailEnable = false; //이메일 사용 가능 여부
+let passwordEnable = false; //패스워드 사용 가능 여부
+function emailCheck(e) {
+  //이메일 확인 함수
+  if (e.type === "focusout" || (e.type === "keyup" && e.key === "Enter")) {
```

Comment:
이런 식으로 구조분해할당을 이용해보셔도 좋을거 같습니다
```suggestion
function emailCheck({ type, key }) {
  //이메일 확인 함수
  if (type === "focusout" || (type === "keyup" && key === "Enter")) {
```